Add buck targets for QNN AOT export

Abhi-hpp opened the pull request:
Summary:
Added the necessary buck targets needed to compile qnn export.
Next steps: Get llama model export to succeed without any errors.

Differential Revision: D61808158

cccclai reviewed (approved):
Looks great! @chiwwang just in case, it's mainly for meta internal buck build as that's the build flow for internal users. open source version (python + cmake) are not affected.

Abhi-hpp committed:
Add buck targets for QNN AOT export (#5476)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/5476

Added the necessary buck targets needed to compile qnn export.
Next steps: Get llama model export to succeed without any errors.

bypass-github-export-checks

Reviewed By: derekxu, cccclai

Differential Revision: D61808158
